typescript/sdk/src/core/EvmCoreModule.ts

```diff
@@ -90,20 +93,104 @@ export class EvmCoreModule extends HyperlaneModule<
     const transactions: AnnotatedEV5Transaction[] = [];
 
     transactions.push(
-      ...this.createMailboxOwnershipTransferTx(actualConfig, expectedConfig),
+      ...(await this.createUpdateDefaultIsmTx(actualConfig, expectedConfig)),
+      ...this.createUpdateMailboxOwnerTx(actualConfig, expectedConfig),
```

Comment:
can we wrap these in a `createUpdateTxs()` fn?
```suggestion
      ...(await this.createUpdateTxs(actualConfig, expectedConfig)),
```

Reply:
Updated to `create*UpdateTx()`

not the biggest fan of the wrapper function. imo, it can be inferred by the individual functions that we are creating update functions. 

also want to reduce unnecessary function nesting, if possible :)

Reply:
> also want to reduce unnecessary function nesting, if possible :)

Why? IMO this is the majority of our current tech debt